.github/workflows/examples.yml

```diff
@@ -0,0 +1,53 @@
+name: Examples
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - uses: ipfs/aegir/actions/cache-node-modules@master
+
+  test-examples:
+    name: Test example ${{ matrix.example.name }}
+    runs-on: ubuntu-latest
+    needs: build
+    continue-on-error: true
+    strategy:
+      matrix:
+        example:
+          - name: js-libp2p-example-chat
+            repo: https://github.com/libp2p/js-libp2p-example-chat.git
+            deps:
+              - '@libp2p/peer-id-factory@$PWD/packages/peer-id-factory'
```

Comment:
where is PWD coming from?

Reply:
From the shell, I think?

Reply:
I didn't know you could do `@<path>` for npm versions...  TIL